corehq/apps/cloudcare/templates/formplayer/menu_list.html

```diff
@@ -71,7 +71,11 @@ <h1 class="page-title"><%- title %></h1>
         </div>
     </div>
     <% if (menuIndex % 2 === 1) { %>
-    <div class="clearfix"></div>
+      <div class="clearfix visible-xs"></div>
```

Comment:
why wouldn't you just have a `<div class="clearfix">` for everything? (i.e. without the if statements)